Lib9c/Action/TransferAsset.cs

```diff
@@ -84,6 +84,17 @@ public override IAccountStateDelta Execute(IActionContext context)
                 throw new InvalidTransferRecipientException(Sender, Recipient);
             }
 
+            if (state.GetState(Addresses.ActivatedAccount) is Dictionary asDict)
```

Comment:
👍

Reply:
AFIK, we split monolithic `ActivatedAccountsState` to many derived states [here](https://github.com/planetarium/lib9c/pull/509). so I think we should check derived address (like [this](https://github.com/planetarium/lib9c/pull/509/files#diff-466f48999a3b02d0666d42edcec40b3f2712742275766d488c60875746134d6bR131-R144)) instead of global activate states. cc @ipdae

Reply:
from `ActivateAccount` https://github.com/planetarium/lib9c/blob/rc-v100077/Lib9c/Action/ActivateAccount.cs#L42